helm/charts/determined/values.schema.json

```diff
@@ -0,0 +1,101 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "properties": {
+      "region": {
+        "type": "string",
+        "enum": [
+          "ORD1",
+          "EWR1",
+          "LAS1",
+          "LGA1"
+        ],
+        "form": true,
+        "title": "Region",
+        "customComponent": {
+          "id": "region_selector",
+          "component": "RegionSelector"
+        }
+      },
+      "resources": {
+        "type": "object",
+        "title": "Resources",
+        "form": true,
+        "properties": {
+          "cpu": {
+            "type": "string",
+            "form":"true",
+            "title": "cpu",
+            "description": "Amount of CPU to provide",
+            "render":"slider",
+            "sliderMin": 8000,
+            "sliderMax": 32000,
+            "sliderUnit": "m",
+            "default": "8000m"
+          },
+          "memory": {
+              "type": "string",
+              "form":"true",
+              "title": "memory",
+              "description": "Amount of Memory to provide",
+              "render":"slider",
+              "sliderMin": 8000,
+              "sliderMax": 32000,
+              "sliderUnit": "Mi",
+              "default": "32000Mi"
```

Comment:
Not sure if it is needed, it will pick up from values file?

Reply:
believe default can just be set in values.yaml